parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -340,6 +344,18 @@ ensureTeleportd() {
     systemctlEnableAndStart teleportd || exit $ERR_SYSTEMCTL_START_FAIL
 }
 
+ensureArtifactStreaming() {
+  systemctl start acr-mirror.service
+  sudo /opt/acr/tools/overlaybd/install.sh
```

Comment:
do we want an exit code if this script fails? or the following script fails?

Reply:
added, thanks!